apicast/src/oauth/keycloak.lua

```diff
@@ -154,7 +154,12 @@ local function parse_and_verify_token(self, jwt_token)
 end
 
 function _M:transform_credentials(credentials)
-  local jwt_obj = parse_and_verify_token(self, credentials.access_token)
+  local jwt_obj, err = parse_and_verify_token(self, credentials.access_token)
+
+    if err then
+      ngx.log(ngx.INFO, err)
```

Comment:
this could be improved to say something meaningful like: 'failed to parse and verify JWT: err'